recorder/server.py

```diff
@@ -0,0 +1,40 @@
+from fastapi import FastAPI, BackgroundTasks, HTTPException
+from record_subscription import run_simulator
+from oedisi.types.common import BrokerConfig
+from fastapi.responses import FileResponse
+import uvicorn
+import socket
+import sys
+import os
+
+app = FastAPI()
+
+@app.get("/")
+def read_root():
+    hostname = socket.gethostname()
+    host_ip = socket.gethostbyname(hostname)
+    return {"hostname": hostname, "host ip": host_ip}
+
+def find_filenames(path_to_dir=os.getcwd(), suffix=".feather" ):
+    filenames = os.listdir(path_to_dir)
+    return [ filename for filename in filenames if filename.endswith( suffix ) ]
+
+@app.get("/download/")
+def read_root():
+    file_list  = find_filenames()
+    if file_list:
+        return FileResponse(path=file_list[0], filename=file_list[0], media_type='text/mp4')
```

Comment:
Why `text/mp4`? Why only the first file? Shouldn't it be the only file or something?

Reply:
there is only one feather file for a given recorder

Reply:
I've added a todo on this, because using text/mp4 is quite strange...